modules/ROOT/pages/spring/spring-module.adoc

```diff
@@ -5,6 +5,20 @@ endif::[]
 
 The Spring module allows Mule apps to make use of the Spring Framework.
 
+== To Install the Spring module
+
+Add it to your application using the Studio palette or add the following dependency in your `pom.xml` file:
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>org.mule.modules</groupId>
+  <artifactId>mule-spring-module</artifactId>
+  <version>1.1.0</version> <!-- or newer -->
```

Comment:
The current Spring Module version is 1.3.0. We also use "later" instead of "newer".
Please change to <version>1.3.0</version>
Spring Module in Exchange: https://www.mulesoft.com/exchange/org.mule.modules/mule-spring-module/

Reply:
Regarding the version update, 1.3.0 is not compatible with Mule 4.1.0 according to Exchange so i'd prefer to avoid updating that until we decide how are we going to handle Connector versioning and compatibilities. 
On the other hand, I can change "newer" to "later"

Reply:
Fixed in d1b2c1e